index.css

```diff
@@ -0,0 +1,526 @@
+}
+
+.humberger-menu a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.humberger-menu a:hover {
+  color: #ec5242;
+}
+
+.top-header {
+  background-image: linear-gradient(rgba(255, 255, 255, 0.9), rgba(240, 240, 240, 1)), url('images/web-bg.jpg');
+  background-position: center;
+  padding: 1.2rem;
+}
+
+.slogan {
+  margin-bottom: 0;
+  color: #ec5242;
+  font-weight: 500;
+  font-size: 1.4rem;
+}
+
+.top-title {
+  color: #ec5242;
+  font-size: 2.1rem;
+  font-weight: 900;
+}
+
+.disc-wrapper {
+  border: 1px solid #fff;
+  margin: 2rem 0;
+  background-color: rgb(245, 245, 245);
+}
+
+.web-description {
+  color: #272a31;
+  padding: 0.8rem;
+}
+
+.time {
+  font-weight: 900;
+  margin-top: 1rem;
+}
+
+.location {
+  color: #272a31;
+  font-weight: 450;
+  padding-bottom: 5rem;
+}
+
+.sec-title {
+  text-align: center;
+}
+
+.programs {
+  background-image: url('./images/programs-bg.jpg');
+  background-size: cover;
+  color: #fff;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.underline {
+  width: 20%;
+  align-self: center;
+  color: #ec5242;
+  opacity: 0.8;
+  margin-top: -0.7rem;
+  border: 1px solid #ec5242;
+}
+
+.program-lists {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.program {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 1rem;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.2);
+  padding: 0.8rem 0.8rem 0.3rem 0.3rem;
+}
+
+.program:hover {
+  border: 2px solid #fff;
+}
+
+.program-description {
+  font-size: 0.8rem;
+  align-self: flex-end;
+}
+
+.program-title {
+  align-self: center;
+  font-size: 1.1rem;
+  font-weight: 600;
+  color: #ec5242;
+}
+
+.top-btn {
+  background-color: #ec5242;
+  width: 80%;
+  border: none;
+  padding: 1.4rem 2rem;
+  color: #fff;
+  align-self: center;
+  margin: 1rem 0;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 1rem;
+}
+
+.speakers h2 {
+  align-self: center;
+  font-weight: 600;
+}
+
+.speaker {
+  display: flex;
+}
+
+.img-wrap {
+  width: 100%;
+  height: auto;
+  padding: 2rem 1rem 0 1rem;
+  margin-top: 1rem;
+  align-self: stretch;
+  background-image: url('./images/black-white.jpg');
+  background-position: top left;
+  background-size: 60%;
+  background-repeat: no-repeat;
+}
+
+.speaker-image {
+  width: 100%;
+  height: auto;
+  position: relative;
+  top: -1.2rem;
+  left: -0.5rem;
+}
+
+.speaker-bio p {
+  font-size: 0.9rem;
+}
+
+.speaker-bio h3 {
+  font-size: 1.1rem;
+  font-weight: 600;
+}
+
+.speaker-title {
+  font-style: italic;
+  color: #ec5242;
+}
+
+.speaker-detail {
+  margin-top: 0;
+}
+
+.fa-angle-down {
+  padding: 1rem;
+  color: #ec5242;
+  font-weight: bold;
+}
+
+.mobile-btn {
+  background: transparent;
+  border: 1px solid #000;
+  text-transform: uppercase;
+}
+
+footer {
+  padding: 1.5rem;
```

Comment:
- [ ] [OPTIONAL] Great job with the design and responsiveness, it would be better if you remove this padding from your footer. Currently, there is a white space around your footers. 😎️

Reply:
Thank you for the feedback @amejid I applied the optional change. Thanks